docs/src/pages/careers/react-engineer-x-charts.md

```diff
@@ -0,0 +1,132 @@
+- We're a **remote** company, we prefer asynchronous communication over meetings.
+
+## About the company
+
+MUI started with Material UI, the most successful React implementation of Google's Material Design.
+We have gained a large following, thanks to our focus on Material Design fidelity, our obsession with details, on offering a large number of components, our community engagement, and by carefully improving the DX.
+Today, countless teams and organizations rely on our open-source libraries to build their design system.
+
+A couple of years ago, we started to expand our suite of products.
+We released [MUI X](https://mui.com/x/), a collection of advanced components; [MUI Design kits](https://mui.com/design-kits/), the MUI components available for Figma, Sketch, etc.; and also host [Templates](https://mui.com/templates/), a set of pre-built UI kits.
+We've seen rapid growth with all of them (2-3x per year) and we have more to come.
+We are building [a low-code platform](https://mui.com/toolpad/) for developers to significantly increase the speed and lower the barrier for creating simple CRUD/dashboard UIs with MUI's libraries.
+
+We are a fully distributed team, spread across multiple time zones.
+We work mainly asynchronously, relying mostly on written collaboration.
+Every contributor has the freedom to define how they work — the rest of us won't know what you're doing day-to-day unless you tell us.
+For additional details about the MUI team and culture, you can check our [careers](https://mui.com/careers/) and [about](https://mui.com/about/) pages and also our [public Handbook](https://mui-org.notion.site/Handbook-f086d47e10794d5e839aef9dc67f324b).
+
+## Why we're hiring
+
+The charts team (part of MUI X) needs your help.
+The component is off to [a great start](https://npm-stat.com/charts.html?package=%40mui%2Fx-charts&from=2021-06-01&to=2023-09-13), however we have only started to scratch the surface for the potential of this component. There is x10 more to build.
+
+To be more specific in what we are aiming to achieve in the short to mid-term:
+
+- A tighter integration with the rest of the MUI libraries including Material UI and the Data Grid
+- Faster iteration cycles compared to popular open-source alternatives such as chart.js and Recharts
+
+Looking a the long term we can likely reach 30% of the downloads of alternative React libraries, considering benchmark data. Developers value having a single touch point, but also one they trust, and one that integrates well in the rest of their app.
+
+We also need help to continue to make the components easier to use, make it more customizable, [improve performance](https://www.causal.app/blog/react-perf), make it more accessible, improve the health of the open-source by engaging and collaborating with the community, guide developers to answers, and just generally being a positive presence in the community.
```

Comment:
Are you sure we need this link to causal?

Reply:
I checked and I wasn't able to find such a blog post from us. We can remove it, it was @oliviertassinari suggestion.

Reply:
I think we can remove it. It makes sense for me for a data grid, but it feels a bit strange for charts.